docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,107 @@
+- `moniker`: This variable sets a custom human-readable name for the node.
+
+### RPC Server Configuration Options
+
+- `laddr`: This variable sets the TCP or UNIX socket address for the RPC server to listen on. Default listening address is on "tcp://127.0.0.1:26657"
+
+- `timeout_broadcast_tx_commit`: This variable sets the time to wait for a transaction to be committed during `/broadcast_tx_commit`. Default timeout is 10s.
+
+### Consensus Configuration Options
+
+- `timeout_propose`: This variable sets the time to wait for a proposal block before prevoting nil. Default timeout is 3s
+
+- `timeout_prevote`: This variable sets the time to wait after receiving +2/3 prevotes for "anything". Default timeout is 1s
+
+- `timeout_precommit`: This variable sets the time to wait after receiving +2/3 precommits for "anything". Default timeout is 1s
+
+- `timeout_commit`: This variable sets the time to wait after committing a block before starting on the new height. Default timeout is 1s
+
+### P2P Configuration Options
+
+- `laddr`: This variable sets the Address to listen for incoming connections. Default listening address is "tcp://0.0.0.0:26656"
+
+- `external_address`: This variable sets the Address to advertise to peers for them to dial. If empty, will use the same port as the laddr, and will introspect on the listener
+
+- `persistent_peers`: This variable sets the Comma-separated list of nodes to connect to and maintain persistent connections. This is a required field for a new node to connect to the kwild network.
+
+- `addr_book_strict`: Toggle for strict address routability rules.
+
+- `max_num_inbound_peers`: This variable sets the Maximum number of inbound peers.
+
+- `max_num_outbound_peers`: This variable sets the Maximum number of outbound peers to connect to, excluding persistent peers.
+
+- `unconditional_peer_ids`: This variable sets the List of node IDs for which connections will be (re)established irrespective of the max connections.
+
+- `allow_duplicate_ip`: This variable sets the Toggle to disable guard against peers connecting from the same IP.
+
+### Mempool Configuration Options
+
+- `size`: This variable sets the Maximum number of transactions in the mempool.
+
+- `max_txs_bytes`: This variable sets the Limit on the total size of all transactions in the mempool.
+
+- `cache_size`: This variable sets the Size of the cache used to filter transactions seen earlier.
+
+### State Sync Configuration Options
+
+- `enable`: This variable sets the Toggle to enable state sync rapid bootstrapping.
+
+- `rpc_servers`: This variable sets the RPC servers for light client verification and state data retrieval.
+
+- `temp_dir`: This variable sets the Temporary directory for state sync snapshot chunks.
+
+- `discovery_time`: This variable sets the Time to spend discovering snapshots before initiating a restore.
+
+- `chunk_request_timeout`: This variable sets the Timeout duration before re-requesting a chunk.
```

Comment:
I think it's really weird that the config uses different cases.  We should really make it all upper or all lower